onecgiar-pr-server/src/result-dashboard-bi/bi-reports/repository/bi-report.repository.ts

```diff
@@ -65,77 +61,49 @@ export class BiReportRepository extends Repository<BiReport> {
     return dataCredentials;
   }
 
-  async getTokenPowerBi(
-    report_name?: string | number,
-    subpage_id?: number | string,
-  ) {
-    let reportsBi: BiReport[] = await this.getReportsBi(subpage_id);
-    reportsBi = reportsBi.filter((report) =>
-      typeof report_name === 'number'
-        ? report.id == report_name
-        : report.report_name == report_name,
-    );
-    if (reportsBi.length > 0) {
-      const datasets: BodyPowerBiDTO[] = [];
-      const reportsId: BodyPowerBiDTO[] = [];
-
+  async getTokenPowerBi(report: any) {
```

Comment:
@yecksin Could you change the type any to one more in line with the need of the parameter? @typescript-eslint/no-explicit-any.

Reply:
It will be done progressively, but not at this moment, because there are many errors